README.md

```diff
@@ -167,13 +167,13 @@ Paperclip is distributed as a gem, which is how it should be used in your app.
 Include the gem in your Gemfile:
 
 ```ruby
-gem "paperclip", "~> 5.0.0"
+gem 'paperclip', '~> 5.0.0'
 ```
 
 Or, if you want to get the latest, you can get master from the main paperclip repository:
 
 ```ruby
-gem "paperclip", git: "git://github.com/thoughtbot/paperclip.git"
+gem 'paperclip', git: 'git://github.com/thoughtbot/paperclip.git'
```

Comment:
The Rails community usually use double-quoted strings in Gemfile : [rails/rails/Gemfile](https://github.com/rails/rails/blob/master/Gemfile)
I have no idea why they do this.